element-hq/element-x-android
# Fix notification avatar

## Type of change


## Content


Use ImageLoader associated to a MatrixClient to be able to load avatars.
Create `ImageLoaderHolder` to share the `ImageHolder` between the one used by `LoggedInAppScopeFlowNode` and the one used by `DefaultNotificationDrawerManager`

## Motivation and context


Fixes #1991 

## Screenshots / GIFs


NA

## Tests



- Log in to a session
- Kill the app
- Receive a message from someone with an avatar
- Observe that now the avatar is displayed in the notification.

## Tested devices

- OS version(s):

## Checklist
